doc/Makefile

```diff
@@ -3,7 +3,7 @@
 
 # You can set these variables from the command line, and also
 # from the environment for the first two.
-SPHINXOPTS    ?=
+SPHINXOPTS    ?= -a
```

Comment:
What does `-a` do ?

Reply:
It rewrites all files even if nothing changes. I was having some caching issues where the sidebar wasn't updating when adding to the TOC. Happy to take this out again.